content/blog/2015-03-30-community-roundup-26.md

```diff
@@ -67,7 +67,7 @@ Jay Garcia spent a lot of time during the beta working on a NES music player wit
 
 ## React Native with Babel and webpack
 
-React Native ships with a custom packager and custom ES6 transforms instead of using what the open source community settled on such as webpack and Babel. The main reason for this is performance – we couldn't get those tools to have sub-second reload time on a large codebase.
+React Native ships with a custom packager and custom ES6 transforms instead of using what the open source community settled on such as [webpack](https://webpack.js.org/) and [Babel](http://babeljs.io/). The main reason for this is performance – we couldn't get those tools to have sub-second reload time on a large codebase.
```

Comment:
Sorry, just a couple more nits which I should have caught earlier. Would be good to use https://babeljs.io/ instead of http://babeljs.io/ (they should probably do a redirect)